pygments/lexers/m.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.lexers.m
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for Cache and GT.M Mumps.
+
+    :copyright: Copyright 2006-2019 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+import re
+
+from pygments.lexer import RegexLexer, words, bygroups
+from pygments.token import *
+
+__all__ = ['MLexer']
+
+class MLexer(RegexLexer):
```

Comment:
Please add a docstring with a link to some page about M/Mumps (look at other lexers for reference).